data-processed/ERDC-SEIR/metadata-ERDC-SEIR.txt

```diff
@@ -0,0 +1,41 @@
+team_name: US Army Engineer Research and Development Center
+team_abbr: ERDC
+institution_affil: DoD
+team_contact: glover.e.george@erdc.dren.mil
+team_funding: US Army Geospatial Task Force
+team_experience: 
+
+model_name: SEIR
+model_abbr: SEIR
+model_output: 
+model_repo:
+
+model_contributors: Michael L. Mayo, Michael A. Rowland,  Matthew D. Parno, Ian Detwiller, Matthew W. Farthing, William P. England
```

Comment:
please add an email for at least one contributor.

Reply:
Updated in master